### DISCUSSION TITLE
Still got ts(2307) error after configured the editor sdk

### DISCUSSION THREAD

**Discussion by AkiraVoid** [yarnpkg/berry#5067] [Q&A] [answered]
I am using vscode and yarn 3. I followed the instructions of editor sdk but my project still report ts(2307): Cannot find module... What should I do?

The project is created by `yarn create next-app --typescript`, there's nothing changed except I installed some dependencies.

**Comment by merceyz** [comment, marked as answer]
You're probably running into https://github.com/yarnpkg/berry/issues/5066.

**Comment by the-spyke** [comment]
I have the similar issues on `yarn@3.3.0`. Current `typescript@4.9.4` doesn't work. Downgrading local package to `~4.8.0` solves the issue with VSCode. But fresh CRA still can't build because of `[eslint] Failed to load config "react-app" to extend from.`

**Reply by merceyz** [reply]
Please check the issues next time, https://github.com/yarnpkg/berry/issues/5125.
